README.md

```diff
@@ -54,6 +54,19 @@ Formbricks is your go-to solution for in-product micro-surveys that will superch
 
 Formbricks helps you apply best practices from data-driven work and experience management to make better business decisions. Ask users as they experience your product - and leverage a significantly higher conversion rate. Gather all insights you can - including partial submissions and build conviction for the next product decision. Better data, better business.
 
+Formbricks is a powerful tool for creating in-product micro-surveys. It allows you to gather valuable insights from your users, enabling you to make data-driven decisions that enhance your product's user experience. With Formbricks, you can create surveys with our no-code editor, choose from a variety of templates, target specific user groups, and much more.
+
+### Table of Contents
```

Comment:
this makes sense :) Please add a screenshot of what this would look like to the PR

Reply:
Added